import-export-cli/cmd/aws/init.go

```diff
@@ -0,0 +1,512 @@
+/*
+*  Copyright (c) WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+*  WSO2 Inc. licenses this file to you under the Apache License,
+*  Version 2.0 (the "License"); you may not use this file except
+*  in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied.  See the License for the
+* specific language governing permissions and limitations
+* under the License.
+*/
+
+package aws 
+
+import (
+	"os"
+	"fmt"
+	"strconv"
+	"os/exec"
+	"bufio"
+
+	"errors"
+	"encoding/json"
+	"io/ioutil"
+	"path/filepath"
+	"github.com/spf13/cobra"
+	
+	"github.com/ghodss/yaml"
+	
+	"github.com/wso2/product-apim-tooling/import-export-cli/box"
+ 
+	"github.com/wso2/product-apim-tooling/import-export-cli/impl"
+	yaml2 "gopkg.in/yaml.v2"
+	
+	v2 "github.com/wso2/product-apim-tooling/import-export-cli/specs/v2"
+ 
+	"github.com/wso2/product-apim-tooling/import-export-cli/utils"
+)
+
+var flagApiNameToGet string		//name of the api to get from aws gateway
+var flagStageName string		//api stage to get  
+var dir string					//dir where the aws init command is executed from
+var path string					//path of the OAS extracted from AWS
+var tmpDir string				//temporary directory created to store the OAS extracted from aws till the project is initialized
+var getRestAPIsCmdOutput string 
+var err error 
+var awsInitCmdForced bool
+var initCmdOutputDir string
+
+//common aws cmd flags 
+var apiGateway string = "apigateway"
+
+//aws cmd Output type
+var outputFlag string = "--output"
+var outputType string = "json"
+
+//cmd_1 
```

Comment:
What is this comment? Can't we remove it?

Reply:
the flags used for the *cmd_1* aiya. should i remove it completely or change it to *getAWSCLIVersionCmd* ?